### PR TITLE
fix: show experiment count and change `start time` unit

### PR DESCRIPTION
**Pull request by keita-determined**
## Description
[WEB-1309], [WEB-1198]

## Test Plan
- Check if experiment count is visible in the new exp table
- Check if `start time` unit abbreviation is correct (`mo` instead of `months` etc)

## Checklist

- [x] Changes have been manually QA'd
- [ ] User-facing API changes need the "User-facing API Change" label.
- [ ] Release notes should be added as a separate file under `docs/release-notes/`.
  See [Release Note](https://github.com/determined-ai/determined/blob/master/docs/release-notes/README.md) for details.
- [ ] Licenses should be included for new code which was copied and/or modified from any external code.

## Ticket

[WEB-1309]: https://hpe-aiatscale.atlassian.net/browse/WEB-1309?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[WEB-1198]: https://hpe-aiatscale.atlassian.net/browse/WEB-1198?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: webui/react/src/pages/F_ExpList/glide-table/TableActionBar.tsx [hunk 1]
```diff
@@ -337,6 +337,9 @@ const TableActionBar: React.FC<Props> = ({
               </Button>
             </Dropdown>
           )}
+          <span className={css.expNum}>
+            {experiments.length.toLocaleString()} experiment{experiments.length > 1 && 's'}
```
Comment (keita-determined) [inline webui/react/src/pages/F_ExpList/glide-table/TableActionBar.tsx:341]: should we add specific locale like `en-US` or use user's computer locale?

**Review by hkang1 — COMMENTED** [review]
Looking good! Couple of things:

The number being reported looks like it's the request limit of 200 instead of the total of 203 in my preview. You can see it in the following screenshot where you see the actual total is 203 in the payload response.
Screenshot 2023-05-31 at 10 32 56 AM

The other thing is that there is a space between the number and the label. (e.g. we want `3mo ago` instead of `3 mo ago`).
Screenshot 2023-05-31 at 10 35 41 AM

File: webui/react/src/pages/F_ExpList/glide-table/TableActionBar.tsx (hunk 1 above)
Comment (keita-determined) [reply webui/react/src/pages/F_ExpList/glide-table/TableActionBar.tsx:341]: made a ticket https://hpe-aiatscale.atlassian.net/browse/WEB-1315

**Comment by keita-determined** [comment]
> The number being reported looks like it's the request limit of 200 instead of the total of 203 in my preview. You can see it in the following screenshot where you see the actual total is 203 in the payload response.

I think the `total` in the API response indicates the number of the entire experiments with the conditions (project id, filter etc), not the number of the experiments the API returns. IIRC, the total is used to calculate pagination, so i think the behavior is correct.

---

> The other thing is that there is a space between the number and the label. (e.g. we want 3mo ago instead of 3 mo ago).

nice catch. fixed

**Comment by hkang1** [comment]
> > The number being reported looks like it's the request limit of 200 instead of the total of 203 in my preview. You can see it in the following screenshot where you see the actual total is 203 in the payload response.
> 
> I think the `total` in the API response indicates the number of the entire experiments with the conditions (project id, filter etc), not the number of the experiments the API returns. IIRC, the total is used to calculate pagination, so i think the behavior is correct.
> 
> > The other thing is that there is a space between the number and the label. (e.g. we want 3mo ago instead of 3 mo ago).
> 
> nice catch. fixed

I started a thread about the discrepancy above. Overall from a user POV, I don't think I would really care about how many entries are loaded, and really want to know how many of experiments match my given search criteria.

**Comment by hkang1** [comment]
the start time column looks good!

**Review by hkang1 — COMMENTED** [review]
just one last thing left: update to the updated design specs

**Comment by keita-determined** [comment]
> just one last thing left: update to the updated design specs

fixed

File: webui/react/src/pages/F_ExpList/glide-table/TableActionBar.tsx [hunk 2]
```diff
@@ -363,6 +363,10 @@ const TableActionBar: React.FC<Props> = ({
               </Button>
             </Dropdown>
           )}
+          <span className={css.expNum}>
+            {Loadable.getOrElse(0, total).toLocaleString()} experiment
+            {Loadable.getOrElse(0, total) > 1 && 's'}
```
Comment (hkang1) [inline webui/react/src/pages/F_ExpList/glide-table/TableActionBar.tsx:368]: **nit refactor:** rewrite to only run `Loadable.getOrElse(0, total)` once?

**Review by hkang1 — APPROVED** [review]
One minor nit but this looks to function as expected!

File: webui/react/src/pages/F_ExpList/glide-table/TableActionBar.tsx (hunk 2 above)
Comment (keita-determined) [reply webui/react/src/pages/F_ExpList/glide-table/TableActionBar.tsx]: fixed

**Pull request merged**